Feature is not working so don't use it (for the moment).

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: linux-2.6.38-alpha/drivers/input/touchscreen/wm831x-ts.c
===================================================================
--- linux-2.6.38-alpha.orig/drivers/input/touchscreen/wm831x-ts.c	2011-01-04 10:06:40.000000000 +0100
+++ linux-2.6.38-alpha/drivers/input/touchscreen/wm831x-ts.c	2011-01-05 15:28:12.000000000 +0100
@@ -137,9 +137,9 @@
 	}
 
 	if (!pen_down) {
-		wm831x_set_bits(wm831x, WM831X_TOUCH_CONTROL_1,
+/*		wm831x_set_bits(wm831x, WM831X_TOUCH_CONTROL_1,
 				WM831X_TCH_CVT_ENA, 0);
-
+*/
 		if (wm831x_ts->pressure)
 			input_report_abs(wm831x_ts->input_dev,
 					 ABS_PRESSURE, 0);
@@ -163,9 +163,9 @@
 	struct wm831x_ts *wm831x_ts = irq_data;
 	struct wm831x *wm831x = wm831x_ts->wm831x;
 
-	wm831x_set_bits(wm831x, WM831X_TOUCH_CONTROL_1,
+/*	wm831x_set_bits(wm831x, WM831X_TOUCH_CONTROL_1,
 			WM831X_TCH_CVT_ENA, WM831X_TCH_CVT_ENA);
-
+*/
 	input_report_key(wm831x_ts->input_dev, BTN_TOUCH, 1);
 	input_sync(wm831x_ts->input_dev);
 
@@ -192,8 +192,8 @@
 		ena |= WM831X_TCH_Z_ENA;
 
 	wm831x_set_bits(wm831x, WM831X_TOUCH_CONTROL_1,
-			WM831X_TCH_X_ENA | WM831X_TCH_Y_ENA | WM831X_TCH_Z_ENA,
-			WM831X_TCH_X_ENA | WM831X_TCH_Y_ENA | ena);
+			WM831X_TCH_CVT_ENA | WM831X_TCH_X_ENA | WM831X_TCH_Y_ENA | WM831X_TCH_Z_ENA,
+			WM831X_TCH_CVT_ENA | WM831X_TCH_X_ENA | WM831X_TCH_Y_ENA | ena);
 
 	return 0;
 }
